src/collections/domain/models/Collection.ts

```diff
@@ -0,0 +1,9 @@
+import { DvObjectOwnerNode } from '../../../core'
+export interface Collection {
+  id: number
+  alias: string
+  name: string
+  affiliation: string
+  description: string
```

Comment:
I believe affiliation and description are not required fields, we could leave them as optional

```suggestion
  affiliation?: string
  description?: string
```